src/Microsoft.AspNetCore.OData/Formatter/ODataInputFormatter.cs

```diff
@@ -8,6 +8,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using Microsoft.AspNet.OData.Adapters;
+using Microsoft.AspNet.OData.Batch;
```

Comment:
No codes changed in this file, but adding a using?

Reply:
@xuzhg There was an extension method `GetODataContentIdMapping()` in the `Microsoft.AspNetCore.OData.Extensions` namespace that was returning null instead of the expected dictionary comprising of Content-ID to Location mapping. An extension method with a similar signature is defined in the `Microsoft.AspNetCore.OData.Batch` namespace where it returns the expected value. After deleting the extension method responsible from the bug from [here](https://github.com/OData/WebApi/pull/2012/files#diff-062764cf9e600d884f48c90b5e3c6100L302), I needed to add this `using` to import the extension method that works as expected